src/test/java/org/opendatakit/validate/ValidateExternalSecondaryInstancesTest.java

```diff
@@ -13,8 +13,17 @@
 
 public class ValidateExternalSecondaryInstancesTest {
     @Test
-    public void supports_xlsforms_value_and_label_refs_in_itemsets_using_secondary_external_instances() throws URISyntaxException {
-        final Path path = getPathOf("external_secondary_instance_non_standard_use.xml");
+    public void supports_xlsforms_value_and_label_refs_in_itemsets_using_secondary_external_instances_xml() throws URISyntaxException {
+        supports_xlsforms_value_and_label_refs_in_itemsets_using_secondary_external_instances("external_secondary_instance_non_standard_use_xml.xml");
+    }
+
+    @Test
+    public void supports_xlsforms_value_and_label_refs_in_itemsets_using_secondary_external_instances_csv() throws URISyntaxException {
+        supports_xlsforms_value_and_label_refs_in_itemsets_using_secondary_external_instances("external_secondary_instance_non_standard_use_csv.xml");
```

Comment:
For some reason this didn't catch my eye when the pattern was introduced in #69 -- what does "non-standard use" refer to?

Reply:
The commit history suggests "non-standard" because it's only "itemset definitions like the ones generated by XLSForm which use `<value ref="name"/>` and `<label ref="label"/>`. I don't think "non-standard" clarifies, so I'll remove it.